libs/ml/tests/unit/layers/PRelu.cpp

```diff
@@ -97,9 +97,10 @@ TYPED_TEST(PReluTest, node_forward_test)  // Use the class as a Node
   placeholder_op_ptr->SetData(data);
 
   fetch::math::SizeType in_size = 50u;
```

Comment:
this assignment
in_size = 50u;
is used 2 times in this file, so it would be great to also have a named constant for the value, m.b. smth like
MIN_ELIGIBLE_INPUT_SIZE or DEFAULT_INPUT_SIZE, you know it better :)